ocaml/idl/datamodel.ml

```diff
@@ -663,6 +663,8 @@ let _ =
     ~doc:"The specified VBD device is not recognized: please use a non-negative integer" ();
   error Api_errors.vm_not_resident_here [ "vm"; "host" ]
     ~doc:"The specified VM is not currently resident on the specified host." ();
+  error Api_errors.vm_not_resident [ "vm" ]
+    ~doc:"The specified VM is not currently resident on any host." ();
```

Comment:
Would `The specified VM is currently not resident on any host.` be better because `not resident` is closer together and is the root of the problem?